### PR TITLE
Add Author firebase credentials to preprod deploy

### PR DESCRIPTION
**Pull request by ajmaddaford**

### CODE REVIEW

**Review by samiwel — APPROVED** [review]
Looks good to me. What's the significance of the `((` double parenthesis syntax vs. the `{{` double curly brace?

**Review by samiwel — COMMENTED** [review]
Also, does this configuration need some kind of `TF_VAR author_enable_auth: TRUE` config? or is Auth going to be disabled in pre-prod?

**Comment by jonnyshaw89** [comment]
@samiwel `author_enable_auth` defaults to true

File: eq.yml [hunk 1]
```diff
@@ -844,6 +844,9 @@ jobs:
       TF_VAR_aws_alb_listener_arn: {{preprod_aws_alb_listener_arn}}
       TF_VAR_survey_launcher_url: https://preprod-new-surveys-launch.eq.ons.digital
       TF_VAR_application_cidrs: [10.30.21.192/28,10.30.21.208/28,10.30.21.224/28]
+      TF_VAR_author_firebase_project_id: ((preprod_author_firebase_project_id))
```
Comment (jonnyshaw89) [inline eq.yml:931]: you need to add these to `secrets.yml.example`

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Comment by ajmaddaford** [comment]
@samiwel `{{` is deprecated in favour of `((` - https://concourse.ci/downloads.html#v320. Using `{{` wraps the value in double quotes. 

I've added a card to the backlog to update the use of `{{` to use `((`. I don't think it should be done as part of this pull request as we need to consider whether quotes are required for each value.

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**